public/static/docs/tutorials/index.md

```diff
@@ -1,18 +0,0 @@
-# Tutorials
```

Comment:
tbh: some kind of an index is useful in this case (and other cases?) - when give a direct link to tutorial it feels strange that it opens the same page as the first section? what do you think, @jorgeorpinel ?

Reply:
This is the only index I could find that definitely seemed unnecessary to me at this point, actually. Please see discussion in https://github.com/iterative/dvc.org/issues/727#issuecomment-571407120 (and the next comment too).

Reply:
To summarize, I agree that

> it feels strange that it opens the same page as the first section

But I also don't want to maintain these bullet lists.

Maybe just strip down these indices to only a single paragraph and a sentence like "Please chose a tutorial in the left navigation side bar or click Next to jump into the first category"?

Reply:
Yep, we can strip the description to the essence of it. A paragraph or two. It does not hurt to mention that there are different kinds of tutorials though. It def tedious to maintain an exact description.

Reply:
OK I've stripped down the indices for tutorials (restored), use-cases, and user-guide 👯

So... #891 was only really for the contribution guide 😋

Reply:
Cool! 😎